rtps-r52/tests/mailbox.c

```diff
@@ -26,3 +32,57 @@ int test_mbox_lsio_loopback(void)
     test_end("test_mbox_lsio_loopback", rc);
     return rc;
 }
+
+int test_mbox_rtps_hpps(int mbox_in, int mbox_out){
+    
+    enum hpsc_mbox_test_rc rc = HPSC_MBOX_TEST_SUCCESS;
+
+    if( mbox_in > HPSC_MBOX_CHANNELS || mbox_out > HPSC_MBOX_CHANNELS ){
```

Comment:
This bounds checking is not correct -- please read my previous review more closely.  First, the function parameters here and the variables assigned the output of `atoi` in the calling function need to be `unsigned` type if we're only going to check upper bounds on their values, otherwise they could be < 0, which is obviously not allowed.  Second, the checks need to be `>= HPSC_MBOX_CHANNELS`, not just `> HPSC_MBOX_CHANNELS` (a classic off-by-one bounds checking error).

Reply:
One small formatting correction here.  As Connor had mentioned before, please follow convention in rest of the code and keep a space between if/else/while/etc, conditionals in (), and braces. E.g., if (argc <= 1) {